docs/source/support/releases.rst

```diff
@@ -0,0 +1,86 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Releases and Versioning
+=======================
+
+.. _versioning:
+
+Versioning
+----------
+
+All DAML components follow `Semantic Versioning <https://semver.org/>`_. In short, this means that there is a well defined "public API" changes or breakages to which are indicated by the version number.
```

Comment:
The structure of the second sentence seems odd. Is it missing a comma after _"public API"_?